pages/Transaction.tsx

```diff
@@ -24,8 +24,8 @@ type Boostagram = {
   podcast: string;
   url: string;
   episode?: string;
-  itemID?: string;
-  ts?: string;
+  itemID?: number;
+  ts?: number;
```

Comment:
what if it is a string? 

those are sent from the podcast players and I don't think we make sure that it's the correct type.

Reply:
Yup, in the Hub we accept both types as it broke a while before as few podcast clients were mixing up string/number for such types, best to add support for both if we want to change this